### PR TITLE
added geo array dumper

### PR DESCRIPTION
**Pull request by norkunas**

### CODE REVIEW

**Comment by willdurand** [comment]
LGTM 👍

**Comment by willdurand** [comment]
cc @geocoder-php/geocoder

File: tests/Geocoder/Tests/Dumper/GeoArrayTest.php [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+/**
+ * @author Jan Sorgalla <jsorgalla@googlemail.com>
+ * @author William Durand <william.durand1@gmail.com>
+ */
+class GeoArrayTest extends TestCase
+{
+    private $dumper;
+
+    public function setUp()
```
Comment (Baachi) [inline tests/Geocoder/Tests/Dumper/GeoArrayTest.php]: should be protected.

File: tests/Geocoder/Tests/Dumper/GeoArrayTest.php [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            'geometry' => array(
+                'type' => 'Point',
+                'coordinates' => array(0, 0)
+            ),
+            'properties' => null
+        );
+
+        $result = $this->dumper->dump($address);
+
+        $this->assertTrue(is_array($result));
```
Comment (Baachi) [inline tests/Geocoder/Tests/Dumper/GeoArrayTest.php]: Should be `assertInternalType``

**Comment by norkunas** [comment]
I wonder why this package doesn't include phpunit in require-dev? Hard to know which phpunit version is used for testing

**Comment by willdurand** [comment]
Great! Thanks @norkunas. One more thing: could you please squash your commits?



> I wonder why this package doesn't include phpunit in require-dev? Hard to know which phpunit version is used for testing

I never considered this useful since, most of the time, we only use common testing stuff that work with all PHPUnit versions.

**Comment by norkunas** [comment]
I'm not familiar with squashing :/ Tried to do "git rebase -i HEAD~2" and now got a merge

**Pull request closed without merging**